src/Messenger.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace WMDE\Fundraising\Frontend;
+
+use Swift_Message;
+use Swift_Mime_Message;
+use Swift_Transport;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class Messenger {
+
+	private $mailTransport;
+	private $failedRecipients = [];
+	private $operatorAddress;
+
+	public function __construct( Swift_Transport $mailTransport, MailAddress $operatorAddress ) {
```

Comment:
The idea of introducing this was to have some kind of wrapper for the SwiftMailer lib. I changed it to what you proposed alternatively.